### ISSUE TITLE
REQUEST - Allow changing window movement speeds

### ISSUE THREAD

**Issue by Bryceshaw06** [pablopunk/SwiftShift#17]
Hello, thank you for your work on this app. It is a great alternative to the same functions in Rectangle Pro. The first thing I noticed when using yours is that the movement speed is much more jerky and fast than that of Rectangle's. If you could look into this and add an option to adjust it, that would be amazing. Thanks!

**Comment by pablopunk** [comment]
hey! Sorry I don't notice a lot of difference between both of those. I see they both move the windows in the same way, except Rectangle Pro resizes the windows faster - which feels unnatural to me as my idea was to follow the mouse cursor (i.e move 20px to the right = resize the window 20px to the right).

### Swift Shift

https://github.com/pablopunk/SwiftShift/assets/4324982/fb891f8b-e8ce-4919-b7eb-37544ea74499


### Rectangle Pro


https://github.com/pablopunk/SwiftShift/assets/4324982/99c3e609-ead8-47e1-99b5-b8a3d4a470f8

---
Is this not your experience? Can you share a bit more of it? Screencasts would be great

**Comment by Bryceshaw06** [comment]
I'm sorry, I just tested again and you're completely right! I don't know how I came to that conclusion. For now I'll assume it was a placebo effect and I'm going crazy, but if I later find out it was a bug I encountered, I'll let you know! Again sorry to bother ya, thanks so much for the reply.

SwiftShift: https://share.cleanshot.com/zkHqk2hD 

Rectangle: https://share.cleanshot.com/RrW15Yh2

When I was testing I did notice another bug (Maybe just a feature I don't like?) but I'll open a separate issue for that one :)
